.github/workflows/android_build.yml

```diff
@@ -0,0 +1,25 @@
+name: Android Build
+
+on: pull_request
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build Project
+        run: ./gradlew assemble
+
+      - name: Run Tests
+        run: ./gradlew test
+
+      - name: Ktlint
+        run: ./gradlew ktlintFormat
```

Comment:
did you try to add the `./gradlew publish` here? What would be necessary to get it to work?

Reply:
don't we publish from gitlab 🙈

Reply:
Yea... but It would be really nice to have everything here, right? Maybe I can find some time to figure out how to do that